optimum/intel/openvino/modeling_diffusion.py

```diff
@@ -409,9 +409,9 @@ def _from_pretrained(
             "tokenizer_2": None,
             "tokenizer_3": None,
             "feature_extractor": None,
-            "image_encoder": None,
-            "safety_checker": None,
         }
+
+        additional_submodels = ["image_encoder", "safety_checker"]
```

Comment:
This doesn't solves the issue as it will just ignore and not load the `safety_checker` for example.
I provide a fix based on how diffusers handle the case of `library_name` being a module in pipelines https://github.com/huggingface/optimum-intel/pull/975#discussion_r1827642118

Reply:
pk, thanks, than I can close my pr?

Reply:
yes, let's try merging the other PR today, it should accelerate per-commit openvino CI back to ~20mins